src/lib/assets/rbac.ts

```diff
@@ -109,3 +166,25 @@ export function storeRoleBinding(name: string): kind.RoleBinding {
     ],
   };
 }
+
+/**
+ * Reads and parses RBAC information from the package.json file under the 'pepr' key.
+ *
+ * @returns {RBACMap | null} - An object representing the RBAC rules from package.json or null if not found.
+ */
+export const readRBACFromPackageJson = (): RBACMap | null => {
```

Comment:
not sure we need to read this in again, we have access to this in the function `allYaml` that calls this function, it is the assets, is there a reason for this?

Reply:
The `readRBACFromPackageJSON` function is used to get the custom RBAC rules from the `package.json`. It is called from the `clusterRole` function in `rbac.ts`, which then merges the custom rules into the generated RBAC rules. 

Currently, we are calling the `clusterRole` function twice - once in the `overridesFile` function and once in the `allYaml` function. The `assets/index.ts` file calls `overridesFile` and `allYaml` separately. But, as part of the `yaml.ts` refactoring we could improve this so that `clusterRole` is only called once. For example, in `assets/index.ts` we could call `clusterRole` once to get the merged list of rbac rules, and then hand that list to both `overrides` and `allYaml`.

Reply:
glad to see we are only reading the module config once again.